pints/_log_likelihoods.py

```diff
@@ -741,6 +752,11 @@ def evaluateS1(self, x):
         """ See :meth:`LogPDF.evaluateS1()`. """
         sigma = np.asarray(x[-self._no:])
 
+        # Calculate log-likelihood
+        L = self.__call__(x)
+        if L == -np.inf:
```

Comment:
I'm not sure what you want here, but there is a method np.isfinite(L) that is True on a slightly wider variety of conditions (i.e. -inf, +inf, NaN)

Reply:
Good shout: I've changed it to `np.isneginf`